tests/phpt/performance_inspections/11_implicit_array_cast_warn.php

```diff
@@ -0,0 +1,48 @@
+@kphp_should_warn
+/variable \$a1 is implicitly converted from array< int > to array< mixed >/
+/variable \$b1 is implicitly converted from array< string > to array< mixed >/
+/variable \$a2 is implicitly converted from array< int > to mixed/
+/variable \$b2 is implicitly converted from array< string > to mixed/
+/variable \$a3 is implicitly converted from array< int > to mixed/
+/variable \$b3 is implicitly converted from array< string > to mixed/
+/variable \$a4 is implicitly converted from array< int > to mixed/
+/variable \$b4 is implicitly converted from array< string > to mixed/
```

Comment:
Maybe, add a more complicated conversion test also? For example:
```
class A {
    /** @var tuple(float, mixed[]) */
    public $a;
}

    (new A)->a = tuple(4, [1,2,3]);

```
Leads to 
expresion <...> is implicitly converted from tuple<int , array< int >> to tuple<float , array< mixed >>